datamol/viz/_viz.py

```diff
@@ -22,29 +24,50 @@ def to_image(
     highlight_bond: List[List[int]] = None,
     outfile: str = None,
     max_mols: int = 32,
-    copy: bool = False,
+    copy: bool = True,
     indices: bool = False,
+    bond_indices: bool = False,
+    stereo_annotations: bool = True,
+    legend_fontsize: int = 16,
```

Comment:
:)

Reply:
Good default is life!

On that note, I want to set `use_svg=True` by default, but it will break backward compat... Any strong opinion?

Reply:
Nope, should be the default. We probably need to forward the `LineWidth` of bonds too as for larger resolution/size the default bond line width is not very good.

Reply:
`use_svg=True` is now default and I also added `bond_line_width` (default same as rdkit) + `mols_size` is now 300 by default instead of 200.